crawler/crawler/connectPG/extract_data.py

```diff
@@ -142,6 +142,8 @@ def extract_metadata_generic(self, dbID: int) -> str:
 
             elif att_name in ["file_modify_date", "file_access_date", "file_create_date"]:
                 date_output = refineData(self._input).refine_date(att_exiftool)
+                if '0000-00-00' in date_output:
```

Comment:
This is okay, but probably always a good idea to tag it with a FIXME/TODO.

Reply:
That's true. I forgot to add the tag.

Reply:
No problem :) Just saw it ;)

Reply:
I added the tag, so I will resolve the conversation